composer.json

```diff
@@ -36,6 +34,15 @@
       "GraphQL\\Examples\\Blog\\": "examples/01-blog/Blog/"
     }
   },
+  "scripts": {
+    "static-analysis": [
+      "rm phpstan.phar || true",
```

Comment:
why not simply use phpstan as a dev dependency? this looks unnecessarily complicated

Reply:
cause phpstan require php 7.1 ;)

Reply:
This lib does [too](https://github.com/webonyx/graphql-php/blob/master/composer.json#L12), hm 🤔?

Reply:
I m not on master ;) but 0.12.x

Reply:
Not sure if I understand correctly :D This is how it should look like 

https://github.com/doctrine/dbal/blob/master/composer.json#L23

And in travis, simply use `script: vendor/bin/phpstan analyse`